docs/modules/user-guide/partials/proc_devfile-registry.adoc

```diff
@@ -0,0 +1,50 @@
+[id="proc_devfile-registry_{context}"]
+= Devfile registry
+
+[role="_abstract"]
+A devfile registry is a service that stores and provides devfile stacks to Kubernetes developer tools like odo, Eclipse Che and the OpenShift Developer Console. Therefore, users can access devfiles through the devfile registry.  Devfile registries are based on the Oracle Cloud Infrastructure (OCI) Specification, and devfile stacks are stored as OCI-artifacts in the registry.
+
+Each devfile stack corresponds to a specific runtime or framework, such as,  NodeJS, Quarkus, and Wildfly. A devfile stack also contains resources, such as, a devfile.yaml, logo, and outerloop resources. These devfile stacks give users a template so they can get started developing with devfiles. For more information on the structure of a devfile registry, see https://github.com/johnmcollier/registry-docs/blob/main/registry-structure.adoc[registry-structure].
+
+== Devfile registry components
+
+A devfile registry is composed of two components, a devfile index server and an OCI registry server.
+
+.Devfile index server
+
+The server, and the resulting container image, does the following:
+
+* Uses the index.json file to obtain metadata about the stacks and samples and hosts this metadata in the registry for tools to consume
+* Bootstraps the OCI registry with the devfile stacks
+* Provides an API to interact with the OCI registry and to retrieve devfile stacks
+
+For more information on the index server, see https://github.com/johnmcollier/registry-docs/blob/main/registry-REST-API.adoc[registry-REST-API].
+
+.OCI registry
+
+The devfile stacks in a devfile registry are stored in an OCI-compatible artifact registry, currently based on the https://github.com/distribution/distribution[reference implementation of an OCI registry].
+
+When a devfile registry is deployed, the stacks belonging to the devfile registry are pushed to the OCI registry as multi-layer artifacts. For more information on the structure of the devfile stacks within the OCI registry, see https://github.com/johnmcollier/registry-docs/blob/main/registry-structure.adoc[registry-structure].
```

Comment:
Any opinion on how many links we have in this doc? I personally think we have too many. I think when there's this many links, it becomes _very easy_ for users to leave our page and fall down a rabbit hole. I wanted a second opinion. If anyone agrees and has an opinion on which links to eliminate, I'm all ears, thanks!